elife/nodejs6.sls

```diff
@@ -3,10 +3,6 @@ nodejs6:
     pkgrepo.managed:
         - name: deb http://deb.nodesource.com/node_6.x {{ distro }} main
```

Comment:
we really need to move off this :scream_cat:

Reply:
one overwhelming thing at a time

Reply:
it's an application problem, rather than an infrastructure one